app/helpers/cloud_volume_helper/textual_summary.rb

```diff
@@ -36,11 +36,11 @@ def textual_size
   end
 
   def textual_bootable
-    {:label => _('Bootable'), :value => @record.bootable.to_s}
+    {:label => _('Bootable'), :value => _(@record.bootable.to_s)}
   end
 
   def textual_status
-    {:label => _('Status'), :value => @record.status.to_s}
+    {:label => _('Status'), :value => _(@record.status.to_s)}
```

Comment:
We are going to have to find a way to get these into the translation catalog if they don't already exist.  (Depends on what the values are here - might need help from @agrare )

Reply:
already exists ✨  (at least available and unavailable, if theres another possible string we'd need to check but pretty sure theyve all already been translated)